packages/curvenote-cli/src/submissions/submit.utils.ts

```diff
@@ -259,12 +259,12 @@ export async function confirmUpdateToExistingSubmission(
     process.exit(1);
   }
 
-  if (opts?.kind) {
+  const kind = existingSubmission?.kind;
+  if (opts?.kind && opts.kind !== kind) {
```

Comment:
This just eliminates the note about ignored `kind` if you are simply re-specifying the correct kind.